maintainers/maintainer-list.nix

```diff
@@ -8247,6 +8247,12 @@
     githubId = 483465;
     name = "Mateusz Wykurz";
   };
+  wulfsta = {
+    email = "wulfstawulfsta@gmail.com";
+    github = "Wulfsta";
+    githubId = 13378502;
+    name = "Wulfsta";
+  };
```

Comment:
Please make this into a separate commit:

```
maintainers: add <name>
```

Reply:
How do I go about doing this?

Reply:
looks like you figured this out, but the easiest way I can think of:

```
git reset HEAD^ -- maintainers/
git add maintainers
git commit -m "maintainers: add <name>"
git rebase -i HEAD^^ # switch order, save
```

Reply:
Yeah I just checked out a new branch following upstream, made the changes there, then reverted the master branch and merged the new one.